src/base/bittorrent/session.cpp

```diff
@@ -155,6 +155,19 @@ namespace
 
         return expanded;
     }
+
+    QStringList findAllFiles(const QString &folderPath)
```

Comment:
Interesting reasoning... For me, the **folder** and **directory** are the same thing.